14-EventToCommandBehavior/src/PrismSample/Views/SimpleExamplePage.xaml.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace PrismSample.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class SimpleExamplePage : ContentPage
+    {
+        public SimpleExamplePage()
+        {
+            InitializeComponent();
+        }
+    }
+}
```

Comment:
See comments for EventArgsConverterExamplePage

Reply:
Done with the latest commit. :)